.github/workflows/rdme-changelogs.yml

```diff
@@ -0,0 +1,18 @@
+name: Sync ReadMe Changelogs 🦉
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  rdme-changelogs:
```

Comment:
Would `readme-changelogs` be more readable? (ditto for filename)

Reply:
`rdme` is the name of the [official cli tool](https://docs.readme.com/main/docs/rdme) for readme.com, and the naming was automatically generated by the tool. I can change this 👍